app/src/main/res/values/strings.xml

```diff
@@ -586,9 +586,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="default_description_language">Default description language</string>
   <string name="delete_helper_make_deletion_toast">Trying to nominate %1$s for deletion</string>
   <string name="delete_helper_show_deletion_title">Nominating for deletion</string>
-  <string name="delete_helper_show_deletion_title_success">Success</string>
+  <string name="delete_helper_show_deletion_title_success">Deleted</string>
   <string name="delete_helper_show_deletion_message_if">Nominated %1$s for deletion.</string>
-  <string name="delete_helper_show_deletion_title_failed">Failed</string>
+  <string name="delete_helper_show_deletion_title_failed">Title could not be deleted</string>
```

Comment:
Please use "Cannot be deleted" or "Could not be deleted" instead. Do not include title here.